src/app/navbar/navbar.component.ts

```diff
@@ -2,13 +2,7 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-navbar',
-  template: `
-  <div class="navbar navbar-default">
-    <ul class="nav navbar-nav">
-      <li><a [routerLink]="['/home']">Home</a></li>
-      <li><a [routerLink]="['/quiz']">quiz</a></li>
-    </ul>
-  </div>
-  `
+  templateUrl: './navbar.component.html',
```

Comment:
Looks like you added a `navbar.component.html` file but it's not in the commit. You may have not added it?